freeipa-client/src/main/java/com/sequenceiq/freeipa/client/model/Host.java

```diff
@@ -81,4 +81,17 @@ public Boolean getHasKeytab() {
     public void setMemberOfRole(List<String> memberOfRole) {
         this.memberOfRole = memberOfRole;
     }
+
+    @Override
+    public String toString() {
+        return "Host{"
```

Comment:
You could also use `String.format("....", dn, krbprincipal,...)`.

You could have a `public static char/String` for the apostrophe instead of retyping.

Reply:
maybe even a helper method `apostropheMe(string)` that just lets you do `apostropheMe(dn)`.

Reply:
There are many other files in cloudbreak which just use the `'`. So I think I am going to leave it as is, at least for now.